package.json

```diff
@@ -29,9 +31,13 @@
     "@babel/core": "^7.13.8",
     "@babel/preset-env": "^7.13.9",
     "@babel/preset-react": "^7.12.13",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.4.1",
     "babel-cli": "^6.26.0",
     "babel-loader": "^8.2.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
```

Comment:
I believe you don't need this package since you'll be using the one from @wojtekmaj instead.